pkg/moov/underwriting_api.go

```diff
@@ -20,6 +20,21 @@ func (c Client) UpsertUnderwriting(ctx context.Context, accountID string, underw
 	return CompletedObjectOrError[Underwriting](resp)
 }
 
+// UpsertUnderwriting adds or updates underwriting information for the given account.
+// Returns the underwriting information for the account.
+func (c Client) UpsertUnderwritingV2507(ctx context.Context, accountID string, underwriting UpsertUnderwriting) (*UnderwritingV2507, error) {
+	resp, err := c.CallHttp(ctx,
+		Endpoint(http.MethodPost, pathUnderwriting, accountID),
+		Version("v2025.07.00"),
```

Comment:
We may want to define versions in moov-go and import them in our go-http library.

https://github.com/moovfinancial/go-http/blob/master/pkg/mhttp/versions.go